syntax = "proto3";

package Arrow;

service ArrowService {
  rpc SayArrow (ArrowRequest) returns (ArrowReply);
}

message ArrowRequest {
  bytes data = 1;
}

message ArrowReply {
  repeated string array_names = 1;
  repeated double array_sums = 2;
}